measurement/validation.py

```diff
@@ -277,33 +287,54 @@ def reset_validated_entries(ids: list) -> None:
     Args:
         ids (list): List of measurement ids to reset.
     """
+    times = []
     update = {"is_validated": False, "is_active": True}
     for _id in ids:
-        Measurement.objects.filter(id=_id).update(**update)
+        current = Measurement.objects.filter(id=_id)
```

Comment:
```suggestion
        current = Measurement.objects.get(id=_id)
        current.is_validated = False
        current.is_active = True
        current.save()
```

Something like this?

Reply:
Yes, this should do it. It's funny, I was not getting any error when trying the function...